presto-main/src/main/java/com/facebook/presto/operator/repartition/LongArrayBlockEncodingBuffer.java

```diff
@@ -72,7 +71,12 @@ public void serializeTo(SliceOutput output)
         serializeNullsTo(output);
 
         if (valuesBufferIndex > 0) {
-            output.appendBytes(valuesBuffer, 0, valuesBufferIndex);
+            try {
+                output.appendBytes(valuesBuffer, 0, valuesBufferIndex);
+            }
+            catch (Exception e) {
+                e.printStackTrace();
```

Comment:
@yingsu00 debugging leftovers?

Reply:
@mbasmanova Yes, I'm sorry about it. I checked there're no other debugging leftovers in this PR. Thanks for catching this!